File: app/src/main/java/ir/touristland/Models/HotelDetails/RoomOptionsItem.java
package ir.touristland.Models.HotelDetails;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class RoomOptionsItem {

    @SerializedName("name")
    private String name;

    @SerializedName("description")
    private String description;

    @SerializedName("value")
    private List<ValueItem> value;

    @SerializedName("key")
    private String key;

    public String getName() {
        return name;
    }

    public String getDescription() {
        return description;
    }

    public List<ValueItem> getValue() {
        return value;
    }

    public String getKey() {
        return key;
    }
}

File: app/src/main/java/ir/touristland/Models/HotelDetails/RoomOptionsItem.kt
package ir.touristland.Models.HotelDetails

import com.google.gson.annotations.SerializedName

class RoomOptionsItem {

    @SerializedName("name")
    val name: String? = null

    @SerializedName("description")
    val description: String? = null

    @SerializedName("value")
    val value: List<ValueItem>? = null

    @SerializedName("key")
    val key: String? = null
}